R/dbConnect__duckdb_driver.R

```diff
@@ -43,25 +45,31 @@ dbConnect__duckdb_driver <- function(drv, dbdir = DBDIR_MEMORY, ...,
                                      debug = getOption("duckdb.debug", FALSE),
                                      read_only = FALSE,
                                      timezone_out = "UTC",
-                                     tz_out_convert = c("with", "force"), config = list(), bigint = "numeric") {
+                                     tz_out_convert = c("with", "force"), config = list(), bigint = NULL) {
   check_flag(debug)
   timezone_out <- check_tz(timezone_out)
   tz_out_convert <- match.arg(tz_out_convert)
 
   missing_dbdir <- missing(dbdir)
   dbdir <- path.expand(as.character(dbdir))
 
+  bigint_type_provided <- !is.null(bigint)
+  if (bigint_type_provided) {
+    check_bigint(bigint)
+    drv@bigint <- bigint
+  } else {
+    bigint <- drv@bigint
+  }
+
   # aha, a late comer. let's make a new instance.
   if (!missing_dbdir && dbdir != drv@dbdir) {
```

Comment:
quick question, since dbdir is set by default to `DBDIR_MEMORY` in the function signature, is there a situation in which it can be missing? Thanks